src/components/MessagesList/MessagesGroup/Message/MessagePart/Reactions.vue

```diff
@@ -57,8 +51,16 @@
 			</div>
 		</NcPopover>
 
+		<!-- all reactions button -->
+		<NcButton v-if="isHovered"
```

Comment:
```suggestion
		<NcButton v-if="hasReactions && showControls"
```

Don't we need to check `hasReactions`?

Reply:
no because Reactions comp is not shown if there is no reaction.